sys/include/ztimer/convert.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
```

Comment:
@kaspar030 "cosmetic" comment: I suppose adding also (C) Inria and (C) FU Berlin is appropriate here, and in similar places applicable places, elsewhere?

Reply:
it's not cosmetic. it's mandatory at least for FUB.

Reply:
@waehlisch yes, "cosmetic" was meant as "non-technical" comment here, obviously.